content/projects/gsoc/2020/project-ideas/machine-learning.adoc

```diff
@@ -4,16 +4,15 @@ title: "Jenkins and Machine Learning Plugins for Data Science"
 goal: "Create a new plugin for integrating Jenkins with one of Machine Learning tools (e.g. Jupyter Python, TensorBoard, or Sacred)"
 category: Plugins
 year: 2020
-status: draft 
-showGoogleDoc: true
+status: published
+sig: pipeline authoring
```

Comment:
Incorrect

Reply:
What is incorrect?

Reply:
```suggestion
sig: pipeline-authoring
```

Review history is still here :) But sorry for not putting it explicitly